src/Truncate/README.md

```diff
@@ -0,0 +1,86 @@
+---
+title: 'Truncate'
+type: 'component'
+components:
+- Truncate
+status: 'New'
+designStatus: 'Done'
+devStatus: 'Done'
+---
+
+A Truncate component can help you crop multiline text. There will be three dots at the end of the text.
+
+### Basic Usage
+
+```jsx live
+  <Truncate lines={2}>
```

Comment:
@PKulkoRaccoonGang Even though `lines={2}` in the "Basic Usage" example on the docs site, I'm seeing it with like 2.5 lines on initial render, until I click "Show code example"; then, it reverts to 2 lines as expected:

<img width="959" alt="image" src="https://user-images.githubusercontent.com/2828721/184356963-f7b1d8cb-8ef6-4771-bf63-6fa1f54838af.png">

<img width="910" alt="image" src="https://user-images.githubusercontent.com/2828721/184357267-fea4b439-bbbe-4044-804b-7fd2467af7e8.png">

Any ideas what might be causing this?

Reply:
This behavior is caused by the width of the container, which is different for each theme. When using truncation by the consumer, there will be no such problems.